### ISSUE TITLE
Outdated dependency uimafit-core;2.2.0-SNAPSHOT

### ISSUE THREAD

**Issue by chmeyer** [dkpro/dkpro-script#7]
Running the example from https://dkpro.github.io/dkpro-script/ yields 

```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
General error during conversion: Error grabbing Grapes -- [unresolved dependency: org.apache.uima#uimafit-core;2.2.0-SNAPSHOT: not found]
```

b/c of the snapshot dependency of uima-fit which has been released a while ago. Will there be a DKPro script release (i.e., a version w/o any snapshot dependencies)?

**Comment by reckart** [comment]
Actually, uimafit 2.2.0-SNAPSHOT should no longer be requested because DKPro Core 1.8.0-SNAPSHOT already uses uimaFIT 2.2.0. If you clear your local `~/.groovy/grapes` folder and re-run, it should work. Once DKPro Core 1.8.0 is released, we may consider doing a DKPro Script release based on that.

**Comment by chmeyer** [comment]
I think the problem is that the DKPro script pom.xml lists uimafit 2.2.0-SNAPSHOT as an explicit dependency. But will try cleaning the folder later.

**Comment by reckart** [comment]
Ok, that can be fixed easily since uimaFIT 2.2.0 is out. Will do a commit for that.

**Comment by reckart** [comment]
@chmeyer a problem with Ivy (which is internally used by Groovy/DKPro Script) is that by default it never tries to update SNAPSHOT versions. This is unlike Maven which checks for more recent versions every once in a while. That is why clearing the local cache (of SNAPSHOTs) sometimes helps resolving problems because it forces the fetching of current SNAPSHOTs.

**Comment by chmeyer** [comment]
OK, the uimaFIT issue is now fixed. And after updating from groovy 1.8.6 to groovy 2.4.6, the script starts to do real work. I next ran into the problem that there doesn't seem to be a dkpro-core-1.8.0-SNAPSHOT anymore on zoidberg. I changed the code example from the GitHub pages to 1.9.0-SNAPSHOT, which worked fine. (I guess it will become much easier when 1.8.0 is released..) Made some small changes to the GitHub page as a PR.
